packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -57,3 +57,20 @@
     }
   }
 }
+
+.amplify-authenticator__column {
```

Comment:
Even though some of these classes are no longer used in this PR in favor of using the our built in primitive options, I haven't deleted them because they are going to be used in the angular and vue PRs to remove authenticator inline styling because these primitives are not yet implemented in these other frameworks.